src/ValidatorKeys.cpp

```diff
@@ -274,7 +270,8 @@ ValidatorKeys::revoke ()
 std::string
 ValidatorKeys::sign (std::string const& data) const
 {
-    return strHex(ripple::sign (publicKey_, secretKey_, makeSlice (data)));
+  return strHex(
+      ripple::sign(keys_.publicKey, keys_.secretKey, makeSlice(data)));
```

Comment:
Nit: We don't have formatting enforced by `clang-format` in this project, but I noticed here in particular that the line starting `return` needs two more spaces to fit the expected formatting.